src/components/tx/SignOrExecuteForm/ExecuteForm.tsx

```diff
@@ -122,21 +123,25 @@ const ExecuteForm = ({
         )}
 
         {/* Error messages */}
-        {isExecutionLoop ? (
-          <ErrorMessage>
-            Cannot execute a transaction from the Safe Account itself, please connect a different account.
-          </ErrorMessage>
-        ) : executionValidationError || gasLimitError ? (
-          <ErrorMessage error={executionValidationError || gasLimitError}>
-            This transaction will most likely fail.{' '}
-            {isNewExecutableTx
-              ? 'To save gas costs, avoid creating the transaction.'
-              : 'To save gas costs, reject this transaction.'}
-          </ErrorMessage>
-        ) : (
-          submitError && (
-            <ErrorMessage error={submitError}>Error submitting the transaction. Please try again.</ErrorMessage>
-          )
+        {hasError && (
+          <div className={css.errorWrapper}>
```

Comment:
Not sure if it's better or worse but if you pass the `className` to the `ErrorMessage` components, you need not create the `hasError` flag. What do you think?

Reply:
Nice, I didn't see that we could already pass a `className` to `ErrorMessage`. Lets do it that way so we can remove `hasError`.